lib/travis/build/script/hack.rb

```diff
@@ -0,0 +1,68 @@
+module Travis
+  module Build
+    class Script
+      class Hack < Php
+        DEFAULTS = {
+          hhvm: 'hhvm', # lts
```

Comment:
Is this a change or incorrect comment? Currently, 'hhvm' gets you the latest release, which might not be an LTS

Reply:
I see this now. I'll look into LTS further.